amivapi/groups/model.py

```diff
@@ -115,7 +115,7 @@ def create_user_lookup_filter(self, user_id):
 
         # Additional lookup: Since group name is unique, you can use it as url
         'additional_lookup': {
-            'url': 'regex("[\w]+")',
+            'url': r'regex("[\w]+")',
```

Comment:
Why is this string prefixed as regex?

Reply:
This is actually changed in a separate pull request (#318).

This change is only a small fix for flake8, so we can merge this branch. (which the other PR requires)

Reply:
And I'm not sure what you mean with "prefixed as regex". In python the *r* makes a string *literal*, i.e. without control characters

Reply:
Ah then I misunderstood the meaning of the _r_ prefix.